configs/mask2former/mask2former_swin-t-p4-w7-224_lsj_8x2_50e_coco_ins.py

```diff
@@ -1,4 +1,4 @@
-_base_ = ['./mask2former_r50_lsj_8x2_50e_coco.py']
+_base_ = ['./mask2former_r50_lsj_8x2_50e_coco_ins.py']
```

Comment:
It is better to keep the original configurations of the panoptic segmentation while making new instance segmentation configurations based on them. Keep the original configuration file name unchanged, and named the new instance segmentation configurations as 'mask2former_r50_lsj_8x2_50e_coco_ins'

Reply:
Is it appropriate to use the configs of panoptic segmengtation for instance segmentation training? We hope we can release competitive models.

Reply:
Changed the names as described.